### ISSUE TITLE
Contao 4 update?

### ISSUE THREAD

**Issue by k-webdesign** [codefog/contao-events_subscriptions#4]
Hey 🤗

Any chance you'll update this for Contao 4 in near future?

**Comment by qzminski** [comment]
Hey Kim, please try the `develop` branch for now, it should be compatible with Contao 4 https://github.com/codefog/contao-events_subscriptions/tree/develop

**Comment by skolarianer** [comment]
I tried with the develop branch (`dev-develop` in Contao Manager) but installation fails with the following error:
```
Exception occured: An error occurred while executing the "contao:install-web-dir" command: 
In ContaoModuleBundle.php line 36:
                                                                
  The module folder "system/modules/_autoload" does not exist.
```

**Comment by qzminski** [comment]
I have just fixed that in dcdb32d.

**Comment by qzminski** [comment]
Version 2.0.0 is out.
